src/vs/workbench/services/editor/browser/editorService.ts

```diff
@@ -749,46 +738,50 @@ export class EditorService extends Disposable implements EditorServiceImpl {
 		}
 
 		// Find target groups for editors to open
-		const mapGroupToEditors = new Map<IEditorGroup, Array<IEditorInputWithOptions>>();
+		const mapGroupToTypedEditors = new Map<IEditorGroup, Array<IEditorInputWithOptions>>();
 		for (const editor of editors) {
 			let typedEditor: IEditorInputWithOptions | undefined = undefined;
 			let group: IEditorGroup | undefined = undefined;
 
 			// Resolve override unless disabled
 			if (editor.options?.override !== EditorOverride.DISABLED) {
-				const [overrideResult, resolvedGroup, resolvedOptions] = await this.doResolveEditor(editor, preferredGroup);
-				if (isEditorInputWithOptions(overrideResult)) {
-					typedEditor = overrideResult;
-				} else {
-					typedEditor = isEditorInputWithOptions(editor) ? editor : { editor: this.createEditorInput(editor), options: resolvedOptions ?? editor.options };
+				const [resolvedEditor, resolvedGroup] = await this.doResolveEditor(editor, preferredGroup);
+
+				if (resolvedEditor === OverrideStatus.ABORT) {
```

Comment:
This attempts to address https://github.com/microsoft/vscode/issues/127210 by implementing `ABORT` support for `openEditors`